testgrid/config.yaml

```diff
@@ -2336,6 +2336,8 @@ test_groups:
   gcs_prefix: kubernetes-jenkins/logs/ci-kubernetes-e2e-gke-gci-new-gci-master-upgrade-cluster-new-parallel
 - name: ci-kubernetes-multicluster-ingress-test
   gcs_prefix: kubernetes-jenkins/logs/ci-kubernetes-multicluster-ingress-test
+- name: kubeflow-periodic-0-5-branch
```

Comment:
You may also need to update https://github.com/kubernetes/test-infra/blob/master/testgrid/config.yaml#L5641

Reply:
thanks! updated